source/lib/cli-run-options.test.ts

```diff
@@ -0,0 +1,136 @@
+import test from 'ava';
+import { Factory, type DeepPartial } from 'fishery';
+import Result from 'true-myth/result';
+import { InvalidArgumentError } from 'commander';
+import Maybe, { type Just } from 'true-myth/maybe';
+import { createCliRunOptions, type CliRunOptions, type CreateCliRunOptions } from './cli-run-options.js';
+
+const createCliRunOptionsFactory = Factory.define<CreateCliRunOptions>(() => {
+    return {
+        versionNumber: undefined,
+        commandOptions: {},
+        changelogPath: ''
+    };
+});
+
+const testCreateCliRunOptionsMacro = test.macro(
+    (t, optionsOverrides: DeepPartial<CreateCliRunOptions>, expected: Result<CliRunOptions, InvalidArgumentError>) => {
+        const options = createCliRunOptionsFactory.build(optionsOverrides);
+
+        const actual = createCliRunOptions(options);
+
+        t.deepEqual(actual, expected);
+    }
+);
+
+test(
+    'createCliRunOptions() returns an Result Error when "unreleased" command option exists and a version number was provided',
```

Comment:
```suggestion
    'createCliRunOptions() returns a Result Error when "unreleased" command option exists and a version number was provided',
```

Reply:
Changed in 1a52b58a885673331cfe170a362ce4cc857b5e31